MaterialLib/SolidModels/SolidsBase.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATERIALLIB_SOLIDMODELS_SOLIDBASE_H_
+#define MATERIALLIB_SOLIDMODELS_SOLIDBASE_H_
+
+#include <memory>
+
+#include "ProcessLib/Deformation/BMatrixPolicy.h"
+#include "ProcessLib/Parameter/Parameter.h"
+
+namespace MeshLib
+{
+class Element;
+}
+
+namespace Solids
+{
+template <int DisplacementDim>
+struct SolidsBase
```

Comment:
No objections. Rubber, if considered elastic, will cool if you stretch it, and heat when you compress it. If it is modelled as viscoelastic (usually more appropriate), it will heat due to dissipative effects regardless of the deformation mode.